packages/frontend/src/pages/state-updates/StateUpdatesStats.tsx

```diff
@@ -0,0 +1,86 @@
+import { PedersenHash } from '@explorer/types'
+import classNames from 'classnames'
+import React, { ReactNode } from 'react'
+import { SimpleLink } from '../common/SimpleLink'
+import { formatHash } from '../formatHash'
+import { formatTime } from '../formatTime'
+
+const formatTimestamp = (timestamp: number) => {
+  const date = new Date(timestamp)
+  const day = date.getUTCDate()
+  const month = date.getUTCMonth()
+  const year = date.getUTCFullYear()
+  const hours = date.getUTCHours()
+  const minutes = date.getUTCMinutes()
+  const timeAgo = formatTime(timestamp)
+  return `${year}-${month}-${day} ${hours}:${minutes} UTC (${timeAgo})`
+}
+
+type StatRowProps = {
+  even: boolean
+  title: string
+  content: ReactNode
+  fontRegular?: boolean
+}
+
+function StatRow({ even, title, content, fontRegular }: StatRowProps) {
+  return (
+    <tr className={classNames(even ? ' bg-grey-200' : ' bg-grey-100')}>
+      <td className="p-1.5 text-right font-bold">{title}</td>
```

Comment:
td -> th

Reply:
Why?

Reply:
because the table represents a key value mapping and keys are kept in headers in tables